kili/mutations/asset/__init__.py

```diff
@@ -289,3 +290,38 @@ def generate_variables(batch):
                                               generate_variables,
                                               GQL_ADD_ALL_LABELED_ASSETS_TO_REVIEW)
         return format_result('data', results[0])
+
+    @Compatible(['v2'])
+    @typechecked
+    def send_back_assets_to_queue(
+            self,
+            asset_ids: List[str]):
+        """Send assets back to queue.
+
+        !!! warning
+            Assets without status REVIEWED will be ignored
+
+        Args:
+            asset_ids: The asset IDs to add to review
+
+        Returns:
+            A result object which indicates if the mutation was successful,
+                or an error message.
+
+        Examples:
+            >>> kili.send_back_to_queue(
+                    asset_ids=[
+                        "ckg22d81r0jrg0885unmuswj8",
+                        "ckg22d81s0jrh0885pdxfd03n"
+                        ],
+        """
+        properties_to_batch = {'asset_ids': asset_ids}
+
+        def generate_variables(batch):
+            return {'where': {'idIn': batch['asset_ids']}}
+
+        results = _mutate_from_paginated_call(self,
+                                              properties_to_batch,
+                                              generate_variables,
+                                              GQL_SEND_BACK_ASSETS_TO_QUEUE)
+        return format_result('data', results[0])
```

Comment:
is there something preventing us to return all the results?

Reply:
it may be done that way in other methods, but we lose some information doing that

Reply:
all mutations return an object with the project ID.
To keep it coherent with before, we still return an object with the project ID if all mutations succeed.
If all mutations succeed, results have all the same objects so we just return the first one.

Reply:
But it would be interesting to think again about the mutations results...
Like giving the ids of assets changed and not the project id for example

Reply:
exactly. Basically the full results variable is something like : data : {data : {id : project_id}}, so not much to return except the project id

Reply:
yes this should be done for v3 imho (probably a list to stay consistent with the input)